improvement: webext `permissions.(request|remove)`: clearer resolve value desciption (#17516)
mdn/content: files/en-us/mozilla/add-ons/webextensions/api/permissions/request/index.md

### Return value

> A [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that will be fulfilled with `true` if the is now granted all the permissions listed in the `permissions` argument, or `false` otherwise.

Reviewer: ```suggestion
A [`Promise`](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise) that is fulfilled with `true` if the extension is now granted all the permissions listed in the `permissions` argument, or `false` otherwise.
```

Author: Yikes
